tonic-examples/src/authentication/server.rs

```diff
@@ -3,12 +3,12 @@ pub mod pb {
 }
 
 use pb::{EchoRequest, EchoResponse};
-use std::collections::VecDeque;
 use tonic::{body::BoxBody, transport::Server, Request, Response, Status, Streaming};
+use tonic_examples::ResponseStream;
 use tower::Service;
 
 type EchoResult<T> = Result<Response<T>, Status>;
-type Stream = VecDeque<Result<EchoResponse, Status>>;
+type Stream = ResponseStream<Result<EchoResponse, Status>>;
```

Comment:
in this case we should just resort to `Pin<Box<dyn Stream<...>>>` this way we don't need to implement a dummy one.